dto/src/main/java/org/jboss/pnc/dto/response/ErrorResponse.java

```diff
@@ -34,10 +35,19 @@
 @JsonDeserialize(builder = ErrorResponse.Builder.class)
 public class ErrorResponse {
 
+    /**
+     * Type of the error.
+     */
     private final String errorType;
 
+    /**
+     * User readable error message.
+     */
     private final String errorMessage;
 
+    /**
+     * Object with more detailed information. This can be null or the object type depends on the error type.
```

Comment:
This can be null, eitherwise the ...

Reply:
eitherwise?

Reply:
` This can be null, otherwise the object ...`?
@alexcreasy maybe your expert opinion?

Reply:
"otherwise" is correct! Eitherwise was made up by a non working portion of my brain :-D